pkg/kubectl/proxy_server.go

```diff
@@ -35,7 +35,7 @@ import (
 const (
 	DefaultHostAcceptRE   = "^localhost$,^127\\.0\\.0\\.1$,^\\[::1\\]$"
 	DefaultPathAcceptRE   = "^/.*"
-	DefaultPathRejectRE   = "^/api/.*/exec,^/api/.*/run,^/api/.*/attach"
```

Comment:
It probably meant to cover the `/run` endpoint in kubelet. But I'm not sure, I'm new to kubernetes, I'm only a little bit familiar with its code since 1.2. :unamused: The one thing is clear - there is no `/run` API endpoint in apiserver.